src/Randomizer.SMZ3.Tracking/AutoTracking/LuaConnector.cs

```diff
@@ -77,13 +77,25 @@ public void SendMessage(EmulatorAction message)
                     Domain = GetDomainString(message.Domain)
                 };
             }
+            // Write memory to the emulator
+            else if (message.Type is EmulatorActionType.WriteBytes)
+            {
+                request = new()
+                {
+                    Action = "write_bytes",
+                    Address = message.Address,
+                    Domain = GetDomainString(message.Domain),
+                    WriteValues = message.WriteValues
+                };
+            }
 
             if (request == null) return;
 
             var msgString = JsonSerializer.Serialize(request) + "\0";
             try
             {
                 _socket.Send(Encoding.ASCII.GetBytes(msgString));
+                //_logger.LogInformation(DateTime.Now.ToLongTimeString() + " Sending " + request.Action);
```

Comment:
Wouldn't be a bad idea to leave this in as Debug

Reply:
Well, these messages come and go very, very frequently (10 per second unless it starts getting bogged down), so it hide other debug statements such as the game state information I print to the debug log, but perhaps a property on the AutoTracker that we could set to enable/disable these that are shared to the two different connectors would be a smarter way of doing this than commenting/uncommenting them or adding/removing them each time.